components/PayoutMethodLabel.tsx

```diff
@@ -27,6 +27,10 @@ export const I18nPayoutMethodLabels = defineMessages({
     id: 'PayoutMethod.Type.Other',
     defaultMessage: 'Other',
   },
+  [PayoutMethodType.STRIPE]: {
```

Comment:
We should ideally re-use the standard `i18nPayoutMethodType` from https://github.com/opencollective/opencollective-frontend/blob/main/lib/i18n/payout-method-type.js here.

Whatever we do, we **never** want people to translate "PayPal" or "Stripe", since they're brands. We want to use fixed, hardcoded strings for them.